homeassistant/components/bluetooth/__init__.py

```diff
@@ -64,6 +66,11 @@
 ]
 
 
+def _get_manager(hass: HomeAssistant) -> BluetoothManager:
+    """Get the bluetooth manager."""
+    return cast(BluetoothManager, hass.data[DATA_MANAGER])
+
```

Comment:
This could have been public to mimic device and entity registries. Then most of the wrappers could be avoided perhaps. Not anything i require, but a thought.

Reply:
I'm trying to keep the underlying implementation from leaking out as much as possible since we have already changed it a few times (heavily with the split of the manager and scanner) and avoid breaking changes (most of the time) since I expect we will have to change it again when we have remote connectables.